dNet/ClientPackets.cpp

```diff
@@ -91,14 +89,16 @@ namespace ClientPackets {
 
 		bitStream.Write(LUWString(receiver, 42));
 
+		// Write the rejected words
+		// There has to be 64 items written so we need to pad the rest with 0s
+		int toWrite = 64;
 		for (auto it : rejectedWords) {
+			if (toWrite <= 0) return;
 			bitStream.Write<uint8_t>(it.first); // start index
 			bitStream.Write<uint8_t>(it.second); // length
+			toWrite--;
 		}
-
-		// Pad out the rest of the packet
-		// The client expects 64 items, so we need to write 64 - rejectedWords.size() empty items
-		for (int i = rejectedWords.size(); 64 > i; i++) {
+		for (toWrite; toWrite <= 0; toWrite--) {
```

Comment:
you dont need the `toWrite;`.  a `;` is sufficient, just evaluates to an empty expression